src/extension/background-script/mnemonic/index.ts

```diff
@@ -16,6 +21,23 @@ class Mnemonic {
     this._hdkey = HDKey.fromMasterSeed(seed);
   }
 
+  deriveLiquidMasterBlindingKey(): string {
```

Comment:
Can these liquid methods be moved to `src/extension/background-script/liquid/index.ts`?

Reply:
Fixed in https://github.com/getAlby/lightning-browser-extension/pull/2583